python/pyarrow/tests/test_array.py

```diff
@@ -3573,3 +3573,28 @@ def test_run_end_encoded_from_buffers():
     with pytest.raises(ValueError):
         pa.RunEndEncodedArray.from_buffers(ree_type, length, buffers,
                                            1, offset, children)
+
+
+@pytest.mark.parametrize(('list_array_type'),
+                         [pa.ListViewArray, pa.LargeListViewArray])
+def test_list_view_from_arrays(list_array_type):
+    values = [[1, 2], [3, 4, 5], [6, None, 7], [8]]
+    offsets = [0, 0, 1, 2, 3]
+    sizes = [2, 0, 3, 3, 1]
+    array = list_array_type.from_arrays(offsets, sizes, values)
+
+    assert array.values.to_pylist() == values
```

Comment:
I would expect that you check `array.to_pylist()` here as well?

Reply:
Good point, I need to update values to be a flat list, not a nested list, in order to get the correct value to assert on.